### DISCUSSION TITLE
How did a spammer get my email from a discussion?

### DISCUSSION THREAD

**Discussion by sam-wheat** [community/community#42861] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I received this spam today which cites an ongoing discussion in it's title.  How did this happen?

From: MichelleCarter3596 <notifications@github.com>Sent: Thursday, December 29, 2022 12:33 AMTo: unoplatform/uno <uno@noreply.github.com>Cc: Sam ; Author <author@noreply.github.com>Subject: Re: [unoplatform/uno] .net 7 library project errors (Discussion #10811)
 

Just bought sexy underwear Wanna see

**Comment by Akash1134** [comment, marked as answer]
Hi @sam-wheat, GitHub is responding to concerns expressed by community members regarding recent spam posts & comments. Thanks for sharing and we appreciate your patience.

**Comment by rilysh** [comment]
When you comment on a discussion, you automatically gets subscribed with that discussion thread. And when a new comment (by someone) will be posted on that discussion thread, GitHub will update you or simply send a email through your default email address. You can unsubscribe from that discussion anytime you want.

You can also unsubscribe from all notifications that GitHub provides.
